components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -298,16 +308,24 @@ - (void)commonMDCFlexibleHeaderViewInit {
   _headerContentImportance = MDCFlexibleHeaderContentImportanceDefault;
   _statusBarHintCanOverlapHeader = YES;
 
+  _minMaxHeightIncludesSafeArea = YES;
   _minimumHeight = kFlexibleHeaderDefaultHeight + kPreIOS11ExpectedStatusBarHeight;
 #if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
   if (@available(iOS 11.0, *)) {
-    // Starting from iOS 11, we should adapt the size of this component to take into account
-    // the new status bar sizes and the "no status bar in landscape" functionality.
-    CGFloat statusBarHeight = [UIApplication mdc_safeSharedApplication].statusBarFrame.size.height;
+    // Starting from iOS 11, the default behavior of this component is to adapt its height to
+    // account for changes in the Safe Area insets. We're using the status bar frame instead of
+    // the safeAreaInsets property because at this point this view's Safe Area hasn't been
+    // calculated by the OS.
+    CGFloat statusBarHeight =
+        CGRectGetMaxY([UIApplication mdc_safeSharedApplication].statusBarFrame);
```

Comment:
I think that all of these types of calls will likely want to move to using a `DeviceTopSafeAreaInset()` call in order to be consistent across hardware.